src/base/xml.cpp

```diff
@@ -13,9 +13,12 @@
 #include "cantera/base/global.h"
 #include "cantera/base/utilities.h"
 
+#include <boost/algorithm/string.hpp>
+
 #include <sstream>
 #include <fstream>
 
+namespace ba = boost::algorithm;
```

Comment:
I'm wondering whether it would be better to just have wrappers in `stringUtils.h/cpp` for the 3 functions that are being used over and over again (`iequals`, `to_lower_copy`, and `trim_copy`), rather than the extra include and namespace declaration in so many files. Thoughts?

Reply:
I started down this exact path with a to_lower_copy wrapper but abandoned it as scope creep when I started hitting more "ba" functions. I do think it would be better to try to corral most of the boost usage into a couple files so I'll try another pass since it may be just those three functions.